corehq/apps/user_importer/importer.py

```diff
@@ -257,8 +256,6 @@ def create_or_update_groups(domain, group_specs):
                     group = group_memoizer.create(domain=domain, name=group_name)
         except ResourceNotFound:
             log["errors"].append('There are no groups on CommCare HQ with id "%s"' % group_id)
-        except MultipleResultsFound:
-            log["errors"].append("There are multiple groups on CommCare HQ named: %s" % group_name)
```

Comment:
After the change in https://github.com/dimagi/commcare-hq/pull/7571/files#diff-af96256b43c9e9882610f6f5dde86a82d55e6f71430fd1b57183222a23b3b08aR185, (which i am not sure if was an oversight) we started using [0] instead of the one() method, which I believe stopped raising exceptions for multiple records for the same name, hence this exception would happen anymore.
@dannyroberts pinging you here, since can't ping Nick P anymore and I see you had a look at that change.

Reply:
Honestly I think the correct behavior would be to still check if there are multiple entries and raise an exception, but I am not sure if we are rolling out that possibility after https://github.com/dimagi/commcare-hq/commit/129f0276cdefb2054368f8383fea2cbaf2fd7cc6

Reply:
Okay yeah, I agree with this. Just looking at the code it seems that this can't happen anymore